pkg/services/sqlstore/migrations/user_mig.go

```diff
@@ -116,6 +116,10 @@ func addUserMigrations(mg *Migrator) {
 
 	// Adds salt & rands for old users who used ldap or oauth
 	mg.AddMigration("Add missing user data", &AddMissingUserSaltAndRandsMigration{})
+
+	mg.AddMigration("Add is_disabled column to user", NewAddColumnMigration(userV2, &Column{
```

Comment:
We probably should add additional clarification somewhere of what that field actually supposed to be there for

Reply:
Yes, this makes sense.